lib/Importer/caffe2.proto

```diff
@@ -126,6 +126,10 @@ message QTensorProto {
   repeated int32 data = 6 [packed = true];
   optional string name = 7;
   optional TensorProto.DataType data_type = 8 [default = INT32];
+  repeated double scales = 9;
+  repeated double biases = 10;
```

Comment:
Is there going to be a separate data type in C2 proto which would signify multiple scales/offsets?